### PR TITLE
migrate our sponsors to nextjs

### PR DESCRIPTION
**Pull request by mrzpn**

### CODE REVIEW

File: frontend/components/sponsorsModal.js [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+        <h1 className={styles.sponsorType}>
+          {sponsors[sponsorName][3]} SPONSOR
+        </h1>
+        <div className={styles.inLine}>
+          <p className={styles.sponsorWebsite}>Website: </p>
+          <Link target="blank" href={sponsors[sponsorName][1]} rel="noopener noreferrer">
+            <a className={styles.sponsorLink}>{sponsors[sponsorName][1]}</a>
+          </Link>
+        </div>
+        <p className={styles.sponsorDescription}>{sponsors[sponsorName][4]} </p>
```
Comment (joanna209) [inline frontend/components/sponsorsModal.js:36]: Remove excess whitespace and increase description font size to match the current website 

image

File: frontend/data/sponsorData.js [hunk 2]
```diff
@@ -0,0 +1,573 @@
...
+    "GOLD",
+    placeHolderDescription(),
+  ],
+  WooliesX: [
+    "WooliesX",
+    "https://wooliesx.com.au/",
+    "WooliesX.png",
+    "GOLD",
+    placeHolderDescription(),
+  ],
```
Comment (joanna209) [inline frontend/data/sponsorData.js:551]: Add logos and descriptions for VMWare and VivCourt

**Review by joanna209 — CHANGES_REQUESTED** [review]

File: frontend/data/sponsorData.js (hunk 2 above)
Comment (mrzpn) [reply frontend/data/sponsorData.js:551]: there are no descriptions in the sponsor doc for VMWare or VivCourt, so I've added placeholder ones

**Review by joanna209 — CHANGES_REQUESTED** [review]
Some small nitpick comments. After all these comments are resolved, should be all good to go! Awesome work so far 👍

**Review by joanna209 — APPROVED** [review]

**Pull request merged**